conda_smithy/utils.py

```diff
@@ -64,6 +64,8 @@ class NullUndefined(jinja2.Undefined):
     def __unicode__(self):
         return self._undefined_name
 
+    __str__ = __unicode__
```

Comment:
I'd argue we invert this and use `__str__` as the method name, with the `__unicode__` alias if we want to keep backwards compatibility, but afaik we are not supporting py2k anymore so I'd just rename `__unicode__` to `__str__` and call it a day.

Reply:
Thanks, that makes sense to me. I agree that maintaining Python2 compatibility is not necessary and have adopted your suggestion.